### PR TITLE
Update BANNO.LOANPAYMENT.SKIP.V1.POW

### PR DESCRIPTION
**Pull request by tkainz**
Added code to clean up special characters from share/loan descriptions and from TOC text.

### CODE REVIEW

File: loan-payment-skip/REPWRITERSPECS/BANNO.LOANPAYMENT.SKIP.V1.POW [hunk 1]
```diff
@@ -454,14 +469,17 @@ PROCEDURE PROCPERFORMSKIPAPAYMENT
          NEWLINE
          PRINT Q+"nextDueDate"+Q+": "+Q+FORMAT("99/99/9999",LNEWDUEDATE)+Q
          NEWLINE
-         FOR EACH SHARE WITH (SHARE:ID=@RGUSERCHR2)
+	       FOR EACH SHARE WITH (SHARE:ID=@RGUSERCHR2)
```
Comment (svetkomir) [inline loan-payment-skip/REPWRITERSPECS/BANNO.LOANPAYMENT.SKIP.V1.POW:472]: Don't mix tabs and spaces pls.

Comment (tkainz) [reply loan-payment-skip/REPWRITERSPECS/BANNO.LOANPAYMENT.SKIP.V1.POW]: corrected - thanks

Comment (tkainz) [reply loan-payment-skip/REPWRITERSPECS/BANNO.LOANPAYMENT.SKIP.V1.POW]: searched for and found a handful more.  fixed

**Comment by christinasimms** [comment]
Can you please confirm you have tested this in Banno mobile and Banno Online?

Also, can you please add this?  I forgot to add it to the Github version.  
**  3/24/2020:  fixes for no fee and skip period.

**Comment by tkainz** [comment]
Added change history note from 3/24. Program was tested in Banno and mobile

**Pull request merged**